packages/manager/src/features/Profile/Settings/Settings.tsx

```diff
@@ -161,3 +167,9 @@ export const ProfileSettings = () => {
     </>
   );
 };
+
+const ThemeKeyboardShortcut = isOSMac ? (
+  <Code>Ctrl + Shift + L</Code>
+) : (
+  <Code>Alt + Shift + L</Code>
+);
```

Comment:
I think I prefer this to just wrapping it all in one `<Code />`

```suggestion
const ThemeKeyboardShortcut = isOSMac ? (
  <span>
    <Code>Ctrl</Code> + <Code>Shift</Code> + <Code>L</Code>
  </span>
) : (
  <span>
    <Code>Alt</Code> + <Code>Shift</Code> + <Code>L</Code>
  </span>
);
```